gnat2goto/driver/range_check.adb

```diff
@@ -27,4 +39,20 @@ package body Range_Check is
                  Bit_Width => Bit_Width);
    end Load_Bound_In_Hex;
 
+   ----------------------------
+   -- Load_Real_Bound_In_Hex --
+   ----------------------------
+
+   function Load_Real_Bound_In_Hex (Index : Integer; Bit_Width : Integer)
+                                    return String
+   is
+      Bound : constant Ureal :=
+        Ureal (Integer_Bounds_Real_Table.Element (Index));
+   begin
+      return (case Bit_Width is
+                 when 32 => Convert_Ureal_To_Hex_32bits_IEEE (Bound),
+                 when 64 => Convert_Ureal_To_Hex_64bits_IEEE (Bound),
+                 when others => raise Program_Error);
```

Comment:
Maybe just have the 32/64 as a precondition to the function? Or even just use a two value enum like `type Float_Format is (IEEE_32_Bit, IEEE_64_Bit);` together with something like 

```
function To_Float_Format (Float_Width : Integer) return Float_Format is
begin
case Float_Width is
when 32 => return IEEE_32_Bit;
when 64 => return IEEE_64_Bit;
when others =>
  // print some error message along the lines of 'We only support 32 and 64 bit floats'
  raise Unknown_Float_Format;
end To_Float_Format;
```
?